### DISCUSSION TITLE
Add server functionality to SPA

### DISCUSSION THREAD

**Discussion by wasiqibnzahid** [quasarframework/quasar#14454] [General - Components / Directives / etc]
Hello, I am working on an SPA using quasar and firebase tools, now I need to add some functionality that only works with firebase admin sdk so I need a server side. As there is no separate server side for my project, I wanted to know if there is a way to integrate nuxtjs into my quasar project or use some other method to run a privileged environment to run the admin sdk. Some way to start a local server side (preferably with nuxt)

**Comment by stefanvanherwijnen** [comment]
Use SSR mode (https://quasar.dev/quasar-cli-vite/developing-ssr/introduction). There is no support for Nuxt (seems te be a common request but someone should create a Nuxt plugin for Quasar support)
